2023/gaussian-integral/french/sentence_translations.json

```diff
@@ -362,7 +362,7 @@
  {
   "input": "We need to add up the areas of all these rectangles, for values of x ranging from negative infinity up to infinity, and the use of that notation dx is kind of meant to imply you shouldn't think of any specific width, but instead you ask, as the chosen width for your rectangles gets thinner and thinner, what does this sum of all those areas approach?",
   "model": "nmt",
-  "translatedText": "Nous devons additionner les aires de tous ces rectangles, pour des valeurs de x allant de moins l'infini à plus l'infini, et l'utilisation de cette notation dx est en quelque sorte censée impliquer que vous ne devriez pas penser à une largeur spécifique quelconque, mais plutôt à une largeur spécifique qui diminue à mesure que la largeur choisie pour vos rectangles devient de plus en plus fine, à quoi se rapproche cette somme de toutes ces aires ?",
+  "translatedText": "Nous devons additionner les aires de tous ces rectangles, pour des valeurs de x allant de moins infini à plus infini, et l'utilisation de cette notation dx est en quelque sorte censée impliquer que vous ne devriez pas penser à une largeur spécifique quelconque, mais plutôt à une largeur spécifique qui diminue à mesure que la largeur choisie pour vos rectangles devient de plus en plus fine, de quoi se rapproche cette somme de toutes ces aires ?",
```

Comment:
I'm not sure about removing the `l'` from `l'infini`. Most of the time, especially when discussing limits, I tend to hear `l'infini` more frequently than just `infini`. What do you think?

Reply:
You're right. I was quite surprised, but it is indeed more common to use `l'infini`.